### PR TITLE
Display when things were updated in words

### PR DESCRIPTION
**Pull request by Floppy**
Like so:

screen shot 2016-10-11 at 12 24 51

### CODE REVIEW

**Comment by Floppy** [comment]
TimeAgo can handle auto-updating the words without reloading the page, but I don't know if that's necessary with our own reloading?

Also, I'm not happy with the Jasmine tests which won't work when the time changes. Need a better solution for those.

File: lib/views/includes/rich-header.erb [hunk 1]
```diff
@@ -2,8 +2,8 @@
 <!-- BOOTSTRAP -->
 <script src='//maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js'></script>
 
-<!-- MOMENT -->
-<script src='https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.6/moment.js'></script>
```
Comment (pezholio) [inline lib/views/includes/rich-header.erb]: Do we only need Moment for the date display then?

**Comment by pezholio** [comment]
I'm happy with this - just a conflict to be sorted

**Comment by Floppy** [comment]
oh yeah. Will update.

**Comment by Floppy** [comment]
Conflicts resolved!

File: lib/views/includes/rich-header.erb (hunk 1 above)
Comment (Floppy) [inline lib/views/includes/rich-header.erb]: It was used all over for date conversion, but that's been replaced, and the chart display is the only place the requires it for the datepicker. It's still included explicitly there.

**Pull request merged**